HealthcareSystem/EventSourcingService/Service/EventStoreExampleService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using EventSourcingService.Model;
+using EventSourcingService.Repository;
+
+namespace EventSourcingService.Service
+{
+    public class EventStoreExampleService : IEventStoreExampleService
+    {
+
+        private readonly IDomainEventRepository<ExampleEvent> _exampleEventRepository;
+
+        public EventStoreExampleService(IDomainEventRepository<ExampleEvent> exampleEventRepository)
+        {
+            _exampleEventRepository = exampleEventRepository;
+        }
+
+        public IEnumerable<ExampleEvent> GetAll()
+        {
+            return _exampleEventRepository.GetAll();
+        }
+
+        public void Add(ExampleEvent exampleEvent)
+        {
+            _exampleEventRepository.Add(exampleEvent);
+        }
+
+        public IEnumerable<ExampleEvent> GetAll(Expression<Func<ExampleEvent, bool>> condition)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
I don't really thing the service example should have this method, since I figure actual event services won't actually expose this method? They'll just use the matching repository method while making reports.